Test in README.md fails

Supplied test does not work

```
laava/test$ make
../src/prepare_annotation.py small/annotation.bed small/reference_names.tsv -o build/small.annotation.txt
Traceback (most recent call last):
  File "laava/test/../src/prepare_annotation.py", line 142, in <module>
    write_annotation_txt(args.output, bed_rows, otr_rows)
  File "laava/test/../src/prepare_annotation.py", line 99, in write_annotation_txt
    with open(out_fname, "w+") as outf:
FileNotFoundError: [Errno 2] No such file or directory: 'build/small.annotation.txt'
make: *** [Makefile:24: build/small.annotation.txt] Error 1
```

--- a/test/Makefile
+++ b/test/Makefile
@@ -1,7 +1,8 @@
 # Local tests: Run the core 3 analysis steps on a downsampled copy of the public PacBio scAAV data
 
 small_inputs = small
-out_small_pfx = build/small
+out_dir = build
+out_small_pfx = $(out_dir)/small
 
 
 .PHONY: fast slow clean
@@ -20,11 +21,12 @@ diffcheck:
 $(out_small_pfx)_AAV_report.html: ../src/create_report.R ../src/report.Rmd $(out_small_pfx).annotation.txt $(out_small_pfx).flipflop_assignments.txt
 	$< $(out_small_pfx) $(small_inputs)/annotation.txt pAV-CMV-GFP $(out_small_pfx).flipflop_assignments.txt
 
-$(out_small_pfx).annotation.txt: ../src/prepare_annotation.py $(small_inputs)/annotation.bed $(small_inputs)/reference_names.tsv
-	$^ -o $@
-
 $(out_small_pfx).flipflop_assignments.txt: ../src/get_flipflop_config.py $(out_small_pfx).tagged.bam $(out_small_pfx).per_read.csv
 	$^ -o $(out_small_pfx)
+
+$(out_small_pfx).annotation.txt: ../src/prepare_annotation.py $(small_inputs)/annotation.bed $(small_inputs)/reference_names.tsv
+	mkdir -p $(out_dir)
+	$^ -o $@
 
 $(out_small_pfx).tagged.bam: ../src/summarize_AAV_alignment.py $(small_inputs)/scaav-cba-egfp.bam $(small_inputs)/annotation.txt
 	$^ $(out_small_pfx) --cpus 4

